index.html

```diff
@@ -129,37 +141,116 @@ <h1 class="text-3xl font-bold text-gray-800 mb-2 text-center">🛡️ Scam & Phi
-        onclick="setExample(3)">₿ Crypto Scam</button>
+        onclick="setExample(3)">₿ <span data-i18n="example4">Crypto Scam</span></button>
       <button
         class="inline-block py-2 px-3 m-1 mr-2 bg-white border border-gray-200 rounded-md text-xs cursor-pointer transition-all hover:bg-indigo-500 hover:text-white hover:border-indigo-500 text-black"
-        onclick="setExample(4)">🎁 Prize Scam</button>
+        onclick="setExample(4)">🎁 <span data-i18n="example5">Prize Scam</span></button>
       <button
         class="inline-block py-2 px-3 m-1 mr-2 bg-white border border-gray-200 rounded-md text-xs cursor-pointer transition-all hover:bg-indigo-500 hover:text-white hover:border-indigo-500 text-black"
-        onclick="setExample(5)">💼 Job Scam</button>
+        onclick="setExample(5)">💼 <span data-i18n="example6">Job Scam</span></button>
       <button
         class="inline-block py-2 px-3 m-1 mr-2 bg-white border border-gray-200 rounded-md text-xs cursor-pointer transition-all hover:bg-indigo-500 hover:text-white hover:border-indigo-500 text-black"
-        onclick="setExample(6)">✅ Legitimate</button>
+        onclick="setExample(6)">✅ <span data-i18n="example7">Legitimate</span></button>
     </div>
   </div>
 
   <div class="text-gray-600 text-sm mt-4">
-    <p>This service is provided for free by <a href="https://bettergov.ph" target="_blank"
+    <p data-i18n="footer">This service is provided for free by <a href="https://bettergov.ph" target="_blank"
         rel="noopener noreferrer">BetterGov.ph</a></p>
   </div>
 
   <script>
+    // Translations
+    const translations = {
+      en: {
+        title: "🛡️ Scam & Phishing Detector",
+        subtitle: "Test your messages for spam and phishing attempts",
+        placeholder: "Paste your email or SMS text here to check if it's spam or phishing...\n\nExample: 'URGENT: Your account has been suspended. Click here to verify your identity immediately!'",
+        analyzeButton: "Analyze Text",
+        examplesTitle: "Try these examples:",
+        example1: "GCash Scam",
+        example2: "Delivery SMS",
+        example3: "Bank Alert",
+        example4: "Crypto Scam",
+        example5: "Prize Scam",
+        example6: "Job Scam",
+        example7: "Legitimate",
+        footer: "This service is provided for free by BetterGov.ph",
+        analyzing: "Analyzing...",
+        confidence: "Confidence",
+        enterText: "Please enter some text to analyze",
+        completeCaptcha: "Please complete the CAPTCHA verification"
+      },
+      tl: {
+        title: "🛡️ Detektor ng Scam at Phishing",
+        subtitle: "Suriin ang inyong mensahe para sa spam at phishing",
+        placeholder: "I-paste dito ang iyong email o SMS upang suriin kung ito ay spam o phishing...\n\nHalimbawa: 'MADALIANG PABATID: Ang iyong account ay nasuspinde. I-click dito upang agad na patunayan ang iyong pagkakakilanlan!'",
```

Comment:
```suggestion
        placeholder: "I-paste dito ang iyong email o SMS text upang suriin kung ito ay spam o phishing...\n\nHalimbawa: 'URGENT: Your account has been suspended. Click here to verify your identity immediately!'",
```

Users might get confused if they have to put actual email address/phone number vs just the text content. Also its probably better to retain the example in English (up to you if you want to retain or not).